5-minute Quickstart Typescript Error Fix

If you are following the quickstart guide, you might notice that after installing Typescript you get the following error after running the command ```tsc --watch -m commonjs -t es5 --emitDecoratorMetadata app.ts```:

```
error TS5023: Unknown compiler option 'emitdecoratormetadata'.
```

In order to solve the problem, I ran ``` npm install -g typescript  ```. Maybe the tutorial should be updated if others are having the same issue.

I had a similar problem where it was using the "wrong" tsc with the following error: error TS5023: Unknown option 'watch'
Fixed by changing tsc to tsc.cmd

Hi, I'm still getting tsc error when I follow to install the tsc and run 

tsc --watch -m commonjs -t es5 --emitDecoratorMetadata app.ts

error TS5023: Unknown option 'watch'

I got the error, too.

```
angular2/src/di/decorators.d.ts(34,13): error TS1003: Identifier expected.
angular2/src/di/decorators.d.ts(37,17): error TS1003: Identifier expected.
angular2/src/di/decorators.d.ts(45,13): error TS1003: Identifier expected.
angular2/src/di/decorators.d.ts(48,17): error TS1003: Identifier expected.
angular2/src/di/decorators.d.ts(56,13): error TS1003: Identifier expected.
angular2/src/di/decorators.d.ts(59,17): error TS1003: Identifier expected.
```

Env:

```
OS: Macintosh; Intel Mac OS X 10_10_4
node -v v0.12.7
npm -v 3.1.2
tsc -v message TS6029: Version 1.5.0-beta
angular2@2.0.0-alpha.31
```

yep,
it gives this error message:
command: tsc --watch -m commonjs -t es5 --emitDecoratorMetadata app.ts
error TS5023: Unknown compiler option 'emitdecoratormetadata'.

--- a/public/docs/js/latest/quickstart.jade
+++ b/public/docs/js/latest/quickstart.jade
@@ -49,6 +49,13 @@
     $ npm install -g typescript@^1.5.0-beta
     $ tsc --watch -m commonjs -t es5 --emitDecoratorMetadata app.ts
 
+.callout.is-helpful
+  p.
+    Windows users: if you get an error that an option is unknown, you are probably running
+    an older version of TypeScript. 
+    See <a href="http://stackoverflow.com/questions/23267858/how-do-i-install-typescript">
+    Stack Overflow: How do I install Typescript</a>
+
 // STEP 3 - Import Angular ##########################
 .l-main-section
   h2#section-transpile 3. Import Angular

